contrib/init/sysvinit-debian/docker.default

```diff
@@ -8,7 +8,7 @@
 #
 
 # Customize location of Docker binary (especially for development testing).
-#DOCKER="/usr/local/bin/docker"
+DOCKERD="/usr/bin/dockerd"
```

Comment:
Sorry, I think this is the last one -- this should still be `/usr/local/bin/dockerd` and should be commented out by default. :innocent: